WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -1027,6 +1027,15 @@ + (TracksEventPair *)eventPairForStat:(WPAnalyticsStat)stat
         case WPAnalyticsStatReaderArticleCommentedOn:
             eventName = @"reader_article_commented_on";
             break;
+        case WPAnalyticsStatReaderArticleCommentLiked:
```

Comment:
New values here need to be added to `WPAnalytics.h` in the `WordPress-iOS-Shared` pod.

I'm so sorry we've done this.

Reply:
😆